flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/AutoClosableProcess.java

```diff
@@ -115,6 +115,22 @@ public void runBlocking(final Duration timeout) throws IOException {
 			}
 		}
 
+		public void runBlockingWithRetry(final int maxRetries, final Duration timeoutOfTry) throws IOException {
```

Comment:
rename `timeoutOfTry` to `attemptTimeout`; an additional global timeout may also be useful.

Reply:
OK